src/NBitcoin/ChainBase.cs

```diff
@@ -95,11 +131,23 @@ public bool SameTip(ChainBase chain)
             return this.Tip.HashBlock == chain.Tip.HashBlock;
         }
 
+        /// <summary>
+        /// Gets the target work required for a network and a specific block height.
+        /// </summary>
+        /// <param name="network">The network to use for work computation.</param>
+        /// <param name="height">The height of the block to get work for.</param>
+        /// <returns>The target work.</returns>
```

Comment:
`required work` here might be little better than `target work`, or `target required work`

Reply:
Sounds good.